spec/controllers/admin/accounts_controller_spec.rb

```diff
@@ -349,18 +425,38 @@ def self.name
         sign_in(create(:user, :admin), scope: :user)
         put(:update, params: {id: account.id, account: {
               default_theming_attributes: {
-                theme_name: 'custom',
                 imprint_link_url: 'http://example.com/new'
+              },
+              paged_entry_template_attributes: {
+                theme_name: 'custom'
               }
             }})
 
-        expect(theming.reload.theme_name).to eq('custom')
-        expect(theming.imprint_link_url).to eq('http://example.com/new')
+        expect(theming.reload.imprint_link_url).to eq('http://example.com/new')
       end
 
-      it 'batch updates widgets of default theming' do
-        theming = create(:theming)
-        account = create(:account, default_theming: theming)
+      it 'updates first paged entry template' do
+        pageflow_configure do |config|
+          config.themes.register(:red)
+          config.themes.register(:yellow)
+          config.themes.register(:green)
+          config.themes.register(:blue)
+        end
+        account = create(:account)
+
+        sign_in(create(:user, :admin), scope: :user)
+        put(:update, params: {id: account.id, account: {
+              paged_entry_template_attributes: {
+                theme_name: 'green'
+              }
+            }})
+
+        expect(Pageflow::EntryTemplate.last.theme_name).to eq('green')
```

Comment:
it's always better to go through the accounts associated entry template than to load the last entry_template and assume it's the one from account